apps/widgets/src/pages/admin-dashboard.tsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from 'react';
+import type { Widget, WidgetStatus } from '@walless/graphql';
+import { qlClient } from '@walless/graphql';
+import { updateWidgetStatus } from '@walless/graphql/mutation';
+import { allWidgets } from '@walless/graphql/query';
```

Comment:
combine it with above import, as `import { allWidgets, updateWidgetStatus } from '@walless/graphql';`